### ISSUE TITLE
Getting docker save error when building using manifest

### ISSUE THREAD

**Issue by akhilkochuveedu** [ruckstack/ruckstack#35]
Hi,

I'm using ruckstack version **0.8.2** to create an installer and I'm using **manifest services** in my ruckstack.conf. When I try to build it, I'm getting the following error:
docker version: **19.03.11**
**Docker images are already present in the host machine.**

```
vagrant@microkube:~/ruckstack/bin/myapp$ sudo ./ruckstack build --project ruckstack.conf --out out/
2020/08/13 07:24:24 Cleaning work directory out/work...
2020/08/13 07:24:25 Creating /home/vagrant/ruckstack/bin/myapp/out/demo-project_0.0.1.installer...
2020/08/13 07:24:25 /root/.ruckstack/cache/helm-v3.2.4-linux-amd64.tar.gz already exists. Not re-downloading
2020/08/13 07:24:25 /root/.ruckstack/cache/rancher/k3s/releases/download/v1.18.6+k3s1/k3s already exists. Not re-downloading
2020/08/13 07:24:26 /root/.ruckstack/cache/rancher/k3s/releases/download/v1.18.6+k3s1/k3s-airgap-images-amd64.tar already exists. Not re-downloading
2020/08/13 07:24:29 Service type: manifest
2020/08/13 07:24:29 Service type: manifest
2020/08/13 07:24:29 Service type: manifest
2020/08/13 07:24:29 Service type: manifest
2020/08/13 07:24:29 Collecting containers...
"docker save" requires at least 1 argument.
See 'docker save --help'.

Usage:  docker save [OPTIONS] IMAGE [IMAGE...]

Save one or more images to a tar archive (streamed to STDOUT by default)
panic: exit status 1

goroutine 1 [running]:
github.com/ruckstack/ruckstack/internal/ruckstack/util.Check(...)
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/internal/ruckstack/util/util.go:18
github.com/ruckstack/ruckstack/internal/ruckstack/builder/artifact.(*Artifact).SaveDockerImages(0xc00007bcc0, 0xc000225e40)
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/internal/ruckstack/builder/artifact/artifact.go:251 +0x326
github.com/ruckstack/ruckstack/internal/ruckstack/builder.Build(0x7fff5423987e, 0xe, 0x7fff54239893, 0x4)
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/internal/ruckstack/builder/builder.go:95 +0xa2f
github.com/ruckstack/ruckstack/cmd/ruckstack/cmd.glob..func1(0x4353c00, 0xc000225d80, 0x0, 0x4)
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/cmd/ruckstack/cmd/build.go:22 +0x56
github.com/spf13/cobra.(*Command).execute(0x4353c00, 0xc000225d40, 0x4, 0x4, 0x4353c00, 0xc000225d40)
	/home/nvoxland/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x4354880, 0x44411a, 0x230de20, 0xc000000180)
	/home/nvoxland/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/home/nvoxland/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/ruckstack/ruckstack/cmd/ruckstack/cmd.Execute()
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/cmd/ruckstack/cmd/root.go:56 +0x31
main.main()
	/home/nvoxland/go/src/github.com/ruckstack/ruckstack/cmd/ruckstack/ruckstack.go:8 +0x20
```
Please let me know if I missed something while building.

Thanks in advance

**Comment by nvoxland** [comment]
I'll take a look

**Comment by nvoxland** [comment]
This should be fixed in 0.9.0
